exercises/concept/wizards-and-warriors/src/test/java/FighterTest.java

```diff
@@ -40,15 +78,15 @@ public void testWizardVulnerable() {
     }
 
     @Test
-    @Tag("task:4")
+    @Tag("task:6")
     @DisplayName("The isVulnerable method of the Wizard returns true by default")
     public void testWizardVulnerableByDefault() {
         Wizard wizard = new Wizard();
         assertThat(wizard.isVulnerable()).isTrue();
     }
 
     @Test
-    @Tag("task:4")
+    @Tag("task:6")
```

Comment:
Looks like the tests were incorrectly tagged, starting with this one. It seems to test task 7 (previously task 5). The next three tests should be tagged with 7, 8, 8 respectively.

Reply:
Done!